test/integration/shoots/logging/seed_logging_stack.go

```diff
@@ -139,6 +139,8 @@ var _ = ginkgo.Describe("Seed logging testing", func() {
 				lokiSts.Spec.Template.Spec.Containers[index].Resources = r
 			}
 		}
+		// Remove the affinity because shoots don't have etcd worker pool
```

Comment:
Seeds also do not have etcd worker pool by default. This is a Gardener landscape installation detail.
```suggestion
```